wallet-new/src/Cardano/Wallet/Kernel/DB/AcidState.hs

```diff
@@ -814,6 +826,7 @@ makeAcidic ''DB [
     , 'cancelPending
     , 'applyBlock
     , 'switchToFork
+    , 'applyHistoricalBlocks
     , 'applyHistoricalBlock
```

Comment:
Perhaps we should even _remove_ the acidic version of `applyHistoricalBlock`?

Reply:
Yes, it looks like a good idea.

Reply:
removed :+1:

Reply:
Note that that would break any existing wallet DBs, but since this is not released yet you can get away with it.

Reply:
Ho! Yes indeed ^.^